src/main/java/org/jenkinsci/plugins/workflow/steps/TimeoutStepExecution.java

```diff
@@ -88,5 +221,21 @@ public String getShortDescription() {
         }
     }
 
+    private static final class EnvironmentExpanderImpl extends EnvironmentExpander {
```

Comment:
I believe I can define a variable available in the timeout block, but I couldn't find how to do that.